custom_components/tahoma/__init__.py

```diff
@@ -33,12 +32,12 @@
 
 from .const import (
     CONF_HUB,
-    CONF_UPDATE_INTERVAL,
     DEFAULT_HUB,
-    DEFAULT_UPDATE_INTERVAL,
     DOMAIN,
     IGNORED_OVERKIZ_DEVICES,
     OVERKIZ_DEVICE_TO_PLATFORM,
+    UPDATE_INTERVAL,
+    UPDATE_INTERVAL_ALL_ASSUMED_STATE,
```

Comment:
```suggestion
    UPDATE_INTERVAL_RTS, # RTS devices have an assumed state only
```

Should we just call it RTS to keep it more Somfy oriented? I understand RTS easier than assumed state. And we can add a comment for other maintainers / core contributors to understand the type.